### PR TITLE
Create boot2docker-status-beta

### PR DESCRIPTION
**Pull request by rogeriopradoj**

### CODE REVIEW

File: Casks/boot2docker-status-beta.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask :v1 => 'boot2docker-status-beta' do
+  version '1.1.0.beta2'
+  sha256 'ce16ce45def874edacc719ff5ef6b79655011adbb3284da13bc3fef7e09dbcd2'
+
+  url "https://github.com/nickgartmann/boot2docker-status/releases/download/#{version}/Boot2Docker.Status-v#{version}.beta2.zip"
```
Comment (vitorgalvao) [inline Casks/boot2docker-status-beta.rb]: Needs a `# github.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

Comment (rogeriopradoj) [inline Casks/boot2docker-status-beta.rb]: Done!

Comment (vitorgalvao) [inline Casks/boot2docker-status-beta.rb]: It isn’t there.

Comment (rogeriopradoj) [inline Casks/boot2docker-status-beta.rb]: Sorry, now it's!

**Pull request merged**